spec/lib/vanagon/driver_spec.rb

```diff
@@ -54,6 +54,40 @@ def create_driver(platform, options = {})
                               'engine' => 'hardware' })
     end
 
+    it 'returns the platform and a scheduler engine when scheduled resources are enabled and an engine is specified' do
+      ENV['VANAGON_SCHEDULED_HARDWARE'] = "true"
+
+      platform = eval_platform('aix-7.1-ppc', <<-END)
+        platform 'aix-7.1-ppc' do |plat|
+          plat.build_host ["pe-aix-71-01", "pe-aix-71-02"]
+          plat.engine "scheduled"
+        end
+      END
+
+      info = create_driver(platform).build_host_info
+
+      expect(info).to match({ 'name'   => 'aix-7.1-ppc',
```

Comment:
Yuck. :frowning_man: I'd really like to not make the naming of systems worse. I know we agreed not to change it, but I was hoping to contain the terrible part of it. Are there other options here or am I just doomed to be sad about our platform tagging and naming?